src/main/scala/rocket/TLB.scala

```diff
@@ -13,7 +13,6 @@ import freechips.rocketchip.tile.{XLen, CoreModule, CoreBundle}
 import freechips.rocketchip.tilelink._
 import freechips.rocketchip.util._
 
-case object PAddrBits extends Field[Int]
```

Comment:
Kind of funny this was declared inside the module that implements _virtual_ memory...

Reply:
lol